### PR TITLE
A0-1243: Reduce amount of logs printed on info level

### PR DESCRIPTION
**Pull request by krzysztofziobro**

### CODE REVIEW

**Review by kostekIV — APPROVED** [review]

File: consensus/src/member.rs [hunk 1]
```diff
@@ -460,7 +460,7 @@ where
             &self.not_resolved_parents,
             &self.not_resolved_coords,
         );
-        info!(target: "AlephBFT-member", "{}", status);
+        debug!(target: "AlephBFT-member", "{}", status);
```
Comment (DamianStraszak) [inline consensus/src/member.rs:463]: This is absolutely crucial log, we cannot turn it off :) 
Note that it is NOT logged on start or end. This is a status log. 

I will leave a few more comments saying "status log", which means they should go back to `INFO`.

File: consensus/src/runway/backup.rs [hunk 2]
```diff
@@ -176,7 +176,7 @@ pub async fn run_loading_mechanism<'a, H: Hasher, D: Data, S: Signature, R: Read
         }
     };
     let next_round_backup: Round = units.len() as Round;
-    info!(target: "AlephBFT-unit-backup", "loaded units from backup. Loaded {:?} units", units.len());
+    debug!(target: "AlephBFT-unit-backup", "loaded units from backup. Loaded {:?} units", units.len());
```
Comment (DamianStraszak) [inline consensus/src/runway/backup.rs:179]: status log, `INFO` is fine

File: consensus/src/runway/backup.rs [hunk 3]
```diff
@@ -192,7 +192,7 @@ pub async fn run_loading_mechanism<'a, H: Hasher, D: Data, S: Signature, R: Read
             return;
         }
     };
-    info!(target: "AlephBFT-unit-backup", "received next round from unit collection: {:?}", next_round_collection);
+    debug!(target: "AlephBFT-unit-backup", "received next round from unit collection: {:?}", next_round_collection);
```
Comment (DamianStraszak) [inline consensus/src/runway/backup.rs:195]: status log, `INFO` is fine

File: consensus/src/runway/collection.rs [hunk 4]
```diff
@@ -248,11 +248,11 @@ impl<'a, H: Hasher, D: Data, MK: Keychain> IO<'a, H, D, MK> {
         {
             warn!(target: "AlephBFT-runway", "unable to send resolved request:  {}", e);
         }
-        info!(target: "AlephBFT-runway", "Finished initial unit collection with status: {:?}", self.collection.status());
+        debug!(target: "AlephBFT-runway", "Finished initial unit collection with status: {:?}", self.collection.status());
```
Comment (DamianStraszak) [inline consensus/src/runway/collection.rs:251]: status log, `INFO` is fine

File: consensus/src/runway/collection.rs [hunk 5]
```diff
@@ -248,11 +248,11 @@ impl<'a, H: Hasher, D: Data, MK: Keychain> IO<'a, H, D, MK> {
...
     }
 
     fn status_report(&self) {
-        info!(target: "AlephBFT-runway", "Initial unit collection status report: status - {:?}, collected starting rounds - {}", self.collection.status(), self.collection.collected_starting_rounds);
+        debug!(target: "AlephBFT-runway", "Initial unit collection status report: status - {:?}, collected starting rounds - {}", self.collection.status(), self.collection.collected_starting_rounds);
```
Comment (DamianStraszak) [inline consensus/src/runway/collection.rs:255]: status log, `INFO` is fine

File: consensus/src/runway/collection.rs [hunk 6]
```diff
@@ -271,7 +271,7 @@ impl<'a, H: Hasher, D: Data, MK: Keychain> IO<'a, H, D, MK> {
                         Some(response) => response,
                         None => {
                             warn!(target: "AlephBFT-runway", "Response channel closed.");
-                            info!(target: "AlephBFT-runway", "Finished initial unit collection with status: {:?}", self.collection.status());
+                            debug!(target: "AlephBFT-runway", "Finished initial unit collection with status: {:?}", self.collection.status());
```
Comment (DamianStraszak) [inline consensus/src/runway/collection.rs:274]: status log, `INFO` is fine

**Review by DamianStraszak — CHANGES_REQUESTED** [review]
Thanks. You can switch back the `status logs` (in comments) to `INFO` but otherwise it's fine.

How does it look like now in logs when a session changes?

**Comment by krzysztofziobro** [comment]
info_logs_blockchain
That's how it looks when running a blockchain example.

**Review by DamianStraszak — APPROVED** [review]
Awesome, this is exactly as I wanted this to look like!

**Pull request merged**